libse/SubtitleFormats/UnknownSubtitle36.cs

```diff
@@ -80,20 +82,21 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
                     continue;
                 }
 
-                if (RegexTimeCodes.Match(line).Success)
+                if (line.Length > 57 && (match = RegexTimeCodes.Match(line)).Success)

```

Comment:
First check if line has probability to match the RegexTimeCodes before trying to match